.github/workflows/unit-test.yml

```diff
@@ -166,8 +162,7 @@ jobs:
 
       - name: Build 🔧
         run: |
-          npm run compile
-          # run additional compilation variants
+          npx lerna run protos
```

Comment:
Do we need to run this separately? https://github.com/open-telemetry/opentelemetry-js/blob/main/experimental/packages/otlp-proto-exporter-base/package.json#L10 the "compile" command is running "protos" by default.

Reply:
maybe not. i'll try without

Reply:
Seems like it wasn't needed